docs/compilation.rst

```diff
@@ -30,14 +30,17 @@ Change directory (``cd``) into ``docker`` directory and run ``./build.sh``. If n
 * CUDA_VERSION - CUDA toolkit version (9 for 9.0 or 10 for 10.0). Default is ``10``. If the version is prefixed with `.` then any value ``XX`` can be passed and the user needs to make sure that Dockerfile.cudaXX.deps is present in `docker/` directory.
 * NVIDIA_BUILD_ID - Custom ID of the build. Default is ``1234``.
 * CREATE_WHL - Create a standalone wheel. Default is ``YES``.
-* BUILD_TF_PLUGIN - Create a DALI TensorFlow plugin as well. Default is ``NO``.
+* BUILD_TF_PLUGIN - Create a DALI TensorFlow plugin wheel as well. Default is ``NO``.
+* PREBUILD_TF_PLUGINS - If prebuild DALI TensorFlow plugin. Default is ``YES``.
```

Comment:
Maybe a bit more info on how these two options interact would be good here?

Reply:
Done